src/components/StyledMarkdown.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import {
+  Dimensions,
+  Linking,
+  Platform,
+  WebView,
+} from 'react-native'
+import EStyleSheet from 'react-native-extended-stylesheet'
+import defaultHTML from '../utils/default-md-html'
+
+import { removeMDTab } from '../utils/remove-md-tab'
+
+const MarkdownIt = require('markdown-it'),
+    md = new MarkdownIt({ linkify: true, breaks: true }).use(removeMDTab)
+
+const WEBVIEW_REF = 'WEBVIEW_REF'
+
+class StyledMarkdown extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { height: 0 }
+  }
+
+  render() {
```

Comment:
Can you place this `render()` after the other functions in this component to keep things consistent?